skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/db/DatabaseSchema.kt

```diff
@@ -86,7 +92,8 @@ class Brevredigering(id: EntityID<Long>) : LongEntity(id) {
     var spraak by BrevredigeringTable.spraak
     var avsenderEnhetId by BrevredigeringTable.avsenderEnhetId
     var saksbehandlerValg by BrevredigeringTable.saksbehandlerValg
-    var redigertBrev by BrevredigeringTable.redigertBrev.writeHashTo(BrevredigeringTable.redigertBrevHash)
+    private var redigertBrev by BrevredigeringTable.redigertBrev.writeHashTo(BrevredigeringTable.redigertBrevHash)
```

Comment:
Planen her er å ha to kolonner midlertidig frem til alle brev er kryptert?
Du kan bruke `oneShotJob` til å tvinge det gjennom på første deploy, må nok modifisere `updateBrevredigeringJson` for å tilpasse den til den nye kolonnen.